### PR TITLE
Feature/refactor cucumber tests

### PR DESCRIPTION
**Pull request by mikeSmith08**

### CODE REVIEW

File: src/test/acceptance/features/enter-name-steps.js [hunk 1]
```diff
@@ -1,26 +1,17 @@
...
-
 const LONG_NAME = 'This name is way too long' +
   'This name is way too long' +
   'This name is way too long' +
   'This name is way too long' + // 100
   'This name is way too long' +
   'This name is way too long' +
   'This name is way too long' +
-  'This name is way too long' + // 200
+  'This name is way too long' + // 200a
```
Comment (YetAnotherMatt) [inline src/test/acceptance/features/enter-name-steps.js]: 200a? :)

File: src/test/acceptance/features/hooks.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+/**
+ * Function to (re)initialise the Selenium driver and the page objects before each Scenario.
+ */
+Before(async function () {
+  pages.initialise()
+})
+
+/**
+ * Function to quite the driver at the end of each Scenario.
```
Comment (YetAnotherMatt) [inline src/test/acceptance/features/hooks.js]: quite -> quit (if you're making other changes)

File: src/test/acceptance/features/enter-name-steps.js (hunk 1 above)
Comment (mikeSmith08) [reply src/test/acceptance/features/enter-name-steps.js]: No idea, I've corrected it on the smoke test branch that I'm currently working on.

File: src/test/acceptance/features/hooks.js (hunk 2 above)
Comment (mikeSmith08) [reply src/test/acceptance/features/hooks.js]: Yep, I'll do that on the other branch too.

Comment (mikeSmith08) [reply src/test/acceptance/features/hooks.js]: Actually, the build seems to be not updating in GitHub again so I'll push these small typos to trigger a new build.

**Pull request merged**